### ISSUE TITLE
Parsing narsese with a term named ï causes an exception

### ISSUE THREAD

**Issue by TonyLo1** [opennars/opennars2#46]
`<i-->any>. or <any-->i>.` cause exception

maybe related to interval handling

**Comment by TonyLo1** [comment]
argh, `<I-->any>. or <any-->I>.`

**Comment by patham9** [comment]
is low priority since i is meant to be the interval-prefix currently. But will look at it once our TODO list is done.

**Comment by patham9** [comment]
will be closed after my next pull request is merged.
